src/main/java/org/opensearch/security/ssl/util/ExceptionUtils.java

```diff
@@ -76,7 +76,7 @@ public static OpenSearchException createJwkCreationException(Throwable cause) {
         return new OpenSearchException("An error occurred during the creation of Jwk: {}", cause, cause.getMessage());
     }
 
-    public static OpenSearchException createTransportClientNoLongerSupportedException() {
-        return new OpenSearchException("Transport client authentication no longer supported.");
+    public static OpenSearchException clusterWrongNodeCertConfigException() {
```

Comment:
> This is breaking change because of renaming of API. When I backport this, I will keep the existing API as such and introduce this as a new API, so that it won't break anything.

Thanks for this comment, while true at the java API level, the security plugin is less strict around public API changes where the primary concern is that version upgrades are not impacted by renames/updates.

This change is safe to backport to the 2.x line.